### PR TITLE
Fix initialUnknowns

### PR DESCRIPTION
**Pull request by lochel**

### CODE REVIEW

File: src/OMSimulatorLib/Variable.h [hunk 1]
```diff
@@ -39,18 +39,32 @@
...
+  bool isIndependent(){return fmi2_causality_enu_independent == causality;}
+
+  // initial attribute
+  bool isExact(){return fmi2_initial_enu_exact == initialProperty;}
+  bool isApprox(){return fmi2_initial_enu_approx == initialProperty;}
+  bool isCalculated(){return fmi2_initial_enu_calculated == initialProperty;}
+
+  bool isInitialUnknown(){return (isOutput() && (isApprox() || isCalculated()))
+                              || (isCalculatedParameter());}
+                           // || (isState() && (isApprox() || isCalculated()))
```
Comment (lochel) [inline src/OMSimulatorLib/Variable.h]: @adeas31 Do you know how to check for states using FMIlibrary?

Comment (adeas31) [inline src/OMSimulatorLib/Variable.h]: As far as I remember you need to deduce it from the `ModelStructure->Derivatives`.

Comment (lochel) [inline src/OMSimulatorLib/Variable.h]: @adeas31 @rfranke I have now a list of all derivatives. The modelDescription contains statements like this for the derivatives: `<Real derivative="1"/>` How can I extract that information using FMIL?

Comment (lochel) [inline src/OMSimulatorLib/Variable.h]: FMIL is sometimes a bit strange. The following seem to work:
```c
fmi2_import_real_variable_t* varReal = fmi2_import_get_variable_as_real(var);
fmi2_import_variable_t* varState = (fmi2_import_variable_t*)fmi2_import_get_real_variable_derivative_of(varReal);
```

**Pull request merged**